controllers/constants/constants.go

```diff
@@ -18,4 +18,8 @@ limitations under the License.
 
 package constants
 
-const CleanupFinalizer = "controllers.submariner.io/cleanup"
+const (
+	CleanupFinalizer = "controllers.submariner.io/cleanup"
+	MetricsGWPort    = "32780"
+	MetricsGNPort    = "32781"
```

Comment:
These should live along side `gatewayMetricsServerPort` and `globalnetMetricsServerPort` (and be `int` for consistency, or convert all to `string` if you find it easier)

Reply:
Moved them together and renamed to make it easy to distinguish between two. Stll went with 1int1 and `string` respectively coz `8080/81` are used as both wile `32780/81` are only used as `string`.